app/helpers/cross_domain_analytics_helper.rb

```diff
@@ -0,0 +1,9 @@
+module CrossDomainAnalyticsHelper
+  def cross_domain_tracker(slug)
+    {
+      'register-to-vote' => 'UA-23066786-5',
+      'accelerated-possession-eviction' => 'UA-37377084-12',
+      'renewtaxcredits' => 'UA-43414424-1', # HMRC, https://www.pivotaltracker.com/n/projects/1261204
```

Comment:
Can we lose this link to the pivotal board. If there is information in there which should be attached to this code it would be better to put that in the commit message or the comment. This code will probably outlive the pivotal board.